docs/sources/explore/trace-integration.md

```diff
@@ -63,6 +63,6 @@ Clicking anywhere on the span row shows span details.
 
 You can navigate from a span in a trace view directly to logs relevant for that span. This is available for Tempo, Jaeger and Zipkin data source at this moment.  their relevant documentation for instruction how to configure this feature.
 
-{{< docs-imagebox img="/img/docs/v74/trace-to-logs.png" class="docs-image--no-shadow" max-width= "600px"  caption="Screenshot of the trace view in Explore with new icon next to the spans" >}}
+{{< docs-imagebox img="/img/docs/explore/trace-to-log-7-4.png" class="docs-image--no-shadow" max-width= "600px"  caption="Screenshot of the trace view in Explore with new icon next to the spans" >}}
```

Comment:
```suggestion
{{< docs-imagebox img="/img/docs/explore/trace-to-log-7-4.png" class="docs-image--no-shadow" max-width= "600px"  caption="Screenshot of the trace view in Explore with icon next to the spans" >}}
```

Reply:
I am wary of calling things "new" in the documentation. In a few releases, it won't be new anymore, but odds are the adjective will still be there.